### DISCUSSION TITLE
Design a scheme for restoring data which has been enhanced by `transforms` in MMCV 设计一个方案，能够还原经过 transform 增强的数据

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#378] [MMEngine]
# Design a scheme for restoring data which has been enhanced by `transforms` in MMCV 
 设计一个方案，能够还原经过 transform 增强的数据 

Design a scheme for restoring data which has been enhanced by `transforms` in MMCV

设计一个方案，能够还原经过 transform 模块增强后数据。

## Task Information

**Repository**：MMEngine

**Difficulty**：Hard

**Points**：30

**Estimated Time Consumption**：14 days

## Task Status

**Claimant's Github ID**：aso538

**Claim Time**：2023年08月14日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by aso538** [comment]
Claim the mission
